test/cypress/integration/HomePage.spec.js

```diff
@@ -0,0 +1,40 @@
+describe('Home Page', () => {
+	beforeEach(() => {
+		cy.visit('')
+		cy.injectAxe()
+	})
+
+	it('should load page', () => {
+		cy.title().should('eq', "Anthony and Vinny's CRA App")
+		cy.get('h1').should('contain', 'Home')
```

Comment:
Same as comment in other spec file

```suggestion
		cy.get('h4').should('contain', 'Home')
```